framer/GestureManager.coffee

```diff
@@ -207,11 +177,13 @@ splitStr = (str) ->
 
 addEventListeners = (target, types, handler) ->
 	splitStr(types).map (type) ->
+		#console.log("hammer.addEventListener", type)
 		Framer.CurrentContext.domEventManager.wrap(target)
```

Comment:
Yeah I know. This is some nasty hacking. The end result is simple though: a list of all dom event listeners so we can manage them, because there is no way in js to ask which listeners are connected to a node, or any nodes.